website/models.py

```
@@ -670,21 +671,19 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def clean(self):
+        """Validate model fields."""
+        super().clean()
+        if self.cve_id:
+            from website.cache.cve_cache import CVE_ID_PATTERN
+
+            if not CVE_ID_PATTERN.match(self.cve_id):
+                raise ValidationError(f"Invalid CVE ID format: {self.cve_id}")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)